examples/pyscf_demo.ipynb

```diff
@@ -0,0 +1,719 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Running pyscf to populate the MolecularData class\n",
+    "\n",
+    "The module run_pyscf.py provides a user-friendly way of running pyscf calculations in FermiLib. The basic idea is that once one generates a MolecularData instance, one can then call pyscf with a specification of certain options (for instance, how much memory to use and what calculations to do) in order to compute things about the molecule, update the MolecularData object, and save results of the calculation. The most common calculations users will want in FermiLib would probably be self-consistent field (aka Hartree-Fock calculations). Our code uses these \"SCF\" calculations compute orbitals, integrals, Hartree-Fock energy, and more. Other common calculations are CISD and FCI calculations which also compute the 1-RDM and 2-RDM associated with their answers, CCSD calculations which also compute the CCSD amplitudes (useful for UCC) and MP2 calculations which can also be used to UCCSD initial guesses. \n",
+    "Note that the \"delete_input\" and \"delete_output\" options indicate whether to save automatically generated pyscf input files or not. To use this plugin, you will need to personally download pyscf.\n",
+    "\n",
+    "Warnings: electronic structure calculations are finicky. They sometimes fail for surprising reasons. See the pyscf documentation for more information or consult and electronic structure theory expert."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Warn: Ipython shell catchs sys.args\n",
+      "Warn: Ipython shell catchs sys.args\n"
```

Comment:
The ProjectQ-Framework follows a high coding standard with mandatory code reviews. It would defeat the purpose of mandatory code reviews, if we accept pull requests with unknown warnings (which could then lead to unknown consequences).

If debugging this error takes too long, then please remove this part and submit it with a separate pull request once it has been fixed.